### DISCUSSION TITLE
Is there a way to get Eddienput to control the first player reliably?

### DISCUSSION THREAD

**Discussion by ElCapitanP** [nirgoren/Eddienput#46] [Technical support]
I am testing out timings and would like to test them against the dummy in SF6. For example, I was testing to see how late I could hit a button and have it still register as part of a combo. If I set the dummy to block after first hit then I can see if it blocks the second, or test reversals, etc. Thanks for any input!

**Comment by ElCapitanP** [comment]
I have tried reordering the controllers in Steam Input. I would definitely be open to using keyboard inputs, saw something in the readme about it but couldn't get it to accept the inputs.

**Comment by nirgoren** [comment]
Hi @ElCapitanP ,

I haven't personally tested with SF6 specifically, but usually you can:
1) Disconnect your physical controller/all physical controllers
2) Start Eddienput
3) Start up the game and then finally (optionally) connect your physical controller
That should result Eddienput's virtual controller taking the role of player 1, and your physical controller can be used for player 2 (or not at all of not needed for your testing).
